python/ray/dashboard/modules/tests/test_head.py

```diff
@@ -41,28 +40,6 @@ async def route_delete(self, req) -> aiohttp.web.Response:
     async def route_view(self, req) -> aiohttp.web.Response:
         pass
 
-    @routes.get("/test/dump")
```

Comment:
why moving this?

Reply:
This handler uses `DataSource`. After `DataSource` became a private implementation for `NodeHead`, this handler can no longer access it. However, this route is used in `test_node.py` and `test_dashboard.py`, so I moved the handler to `node_head.py`. Honestly, I think this handler and its related tests follow a bad pattern, because `test_node.py` should only be testing `node_head.py`, not `test_head.py`. One drawback is that this route is now exposed, whereas it wasn't before. Maybe we can add an environment variable to detect whether we're in a test environment. Are there an existing environement variable?